### ISSUE TITLE
(1.8.0+) Backfills Overview page has no "Run Status" visualization

### ISSUE THREAD

**Issue by brunobento-delfos** [dagster-io/dagster#24663]
### Dagster version

1.8.8

### What's the issue?

![image](https://github.com/user-attachments/assets/a728416b-0e91-4a67-9a6b-d192f94a88f5)

I really miss having this view on my Overview > Backfills page for partitioned jobs backfills.
Ever since the 1.8 version, this view is gone, like below:

![image](https://github.com/user-attachments/assets/4fe1382b-7387-4f30-87a1-57b8b112fad3)

It was really helpful to have that view since we can have several backfills running at once for a single instance of Dagster. I hope this gets fixed on future versions :~

### What did you expect to happen?

I wish I could see easily the status of each partition run on the Overview > Backfills page for my partitioned jobs.

### How to reproduce?

just install dagster > 1.8 and dagster < 1.8, and run a backfill based on the job below:
```py
from dagster import (
    DailyPartitionsDefinition,
    Definitions,
    asset,
    define_asset_job,
)

daily_partitions = DailyPartitionsDefinition(
    start_date="2023-01-01", timezone="utc"
)

@asset(
    partitions_def=daily_partitions,
)
def sleep_asset():
    import time
    time.sleep(60)

job = define_asset_job(name="job", selection=[sleep_asset])

defs = Definitions(
    assets=[sleep_asset],
    jobs=[job],
)
```

### Deployment type

Dagster Helm chart

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by salazarm** [comment]
We're redoing our backfills page and have an experimental version of it shipping this week. I'll post here when that goes out.

**Comment by brunobento-delfos** [comment]
Thanks @salazarm ! looking forward to seeing this 😍

**Comment by krukowski** [comment]
@brunobento-delfos the new runs page is available in 1.8.10 -- you can check out [this discussion](https://github.com/dagster-io/dagster/discussions/24898) for an overview. I'm going to mark this as closed because I believe that addresses your concern but feel free to reopen or leave some feedback on that discussion if it doesn't.
